pegasus/sites.v3/code.org/views/carousel_ai_teacher_flexible.haml

```diff
@@ -0,0 +1,24 @@
+- video_ai_teacher_index = 1
+- video_id = ["fireside_chat"]
+- video_ai_teacher_first = ["ik70A1ge-aY"]
+
+.carousel-wrapper#collections_carousel
+  .carousel.carousel--collection
+    .slides
+      - video_ai_teacher_index.times do |index|
+        .slide
+          .action-block.action-block--one-col.flex-space-between
+            .video__container
+              = view :display_video_thumbnail, id: video_id[index], video_code: video_ai_teacher_first[index], play_button: 'center', letterbox: 'false'
+            
+  .pagination
+    #collections_carousel-pagination
+  .dd__buttons
+    %button.carousel__prev#prev_1
+      %img{src: "/images/carousel/prev.svg", alt: "previous"}
+
+    %button.carousel__next#next_1
+      %img{src: "/images/carousel/next.svg", alt: "next"}
+
+
+
```

Comment:
Nit: extra whitespace — there can just be one line of whitespace at the end of the file